crates/hir-def/src/lib.rs

```diff
@@ -476,20 +476,54 @@ impl_from!(
     for ModuleDefId
 );
 
-// FIXME: make this a DefWithBodyId
```

Comment:
Can we give this a comment describing what construct this is? I am not quite sure what an anon const represents personally